services/movie_session.py

```diff
@@ -0,0 +1,46 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(movie_show_time: datetime.time,
+                         movie_id: int,
+                         cinema_hall_id: int) -> None:
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    movie = Movie.objects.get(id=movie_id)
+    MovieSession.objects.create(show_time=movie_show_time,
+                                cinema_hall=cinema_hall,
+                                movie=movie)
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    if session_date:
+        date_obj = datetime.strptime(session_date, "%Y-%m-%d")
+        query = MovieSession.objects.filter(show_time__year=date_obj.year)

```

Comment:
you can use `show_time__date` instead of 3 lines

Reply:
Done.